Tests/Directory.Build.targets

```diff
@@ -4,11 +4,29 @@
     <LangVersion>9.0</LangVersion>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
   </PropertyGroup>
+
+  <PropertyGroup Condition="$(BUILD_REASON) != '' ">
+    <CollectCoverage>true</CollectCoverage>
+    <CoverletOutput>$(MSBuildThisFileDirectory)\..\coverage\</CoverletOutput>
+    <Include>[SonarScanner.*]*</Include>
+    <MergeWith>$(MSBuildThisFileDirectory)\..\coverage\coverage.net48.json</MergeWith>
```

Comment:
Isn't this `.net48` infix the reason why we lost coverage when we moved the UTs to .NET 5?

Does this file even need to contain the TFM? All the UTs target a single TFM and there's no reason for more. Or is it autogenerated, because coverlet does it like that?

Reply:
It's autogenerated.